src/client/ui/github-configuration.hbs

```diff
@@ -5,6 +5,7 @@
     <meta http-equiv='X-UA-Compatible' content='IE=edge' />
     <meta name='ap-local-base-url' content='{{localBaseUrl}}' />
     <title>{{title}}</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/aui/9.4.0/aui/aui-prototyping.min.css" integrity="DTM1Q+8lU7SzJT+FWr0JFisCSZlwfM0GiAKYy7h1s9vIKa/CIh37s9NuOCqIOgK4tmqrjLK4NuWuIPUQNsikHA==" crossorigin="anonymous" referrerpolicy="no-referrer" />
```

Comment:
I can see now why we had to add above changes to our CSP.

Reply:
Yeah, it makes it much easier for us.  Actually, I wonder if we can make this non-versioned and just use the latest version at all times?  That way we don't need to ever update AUI.

Actually you can call `GET https://api.cdnjs.com/libraries/aui` and find the latest one.  Maybe we want to call that periodically (every X minutes) to update the value that we use in the handlebars file?

Reply:
It introduces a risk of getting affected by breaking changes (or bugs)

Btw, is there any better way of versioning it? (Maybe importing it via npm and hosting it in our service?) Can you suggest what's the best practice here?

Reply:
It's css, I don't think it'll cause issues and even if it did, I'd bet you that it would take less time fixing it than it would take to constantly update the version.  I'd rather not have to think about it if I don't need to.

Reply:
This library has also scripts, there is a <scrpit> tag below